gcd/src/test/java/org/spine3/sample/gae/ApplicationShould.java

```diff
@@ -20,13 +20,15 @@
 
 package org.spine3.sample.gae;
 
+import org.junit.Ignore;
 import org.junit.Test;
 
 @SuppressWarnings("InstanceMethodNamingConvention")
 public class ApplicationShould {
 
     private static final String[] ARGS = new String[0];
 
+    @Ignore
     @Test
```

Comment:
I think it's not a good solution to use `@Ignore` with out comment why this test is ignored now.